python/ray/tune/logger.py

```diff
@@ -239,9 +239,10 @@ def flush(self):
     def close(self):
         if self._file_writer is not None:
             if self.trial and self.trial.evaluated_params and self.last_result:
+                flat_result = flatten_dict(self.last_result, delimiter="/")
```

Comment:
Not necessary for this PR, but for all the flattening/unflattening and nested mapping stuff, I'd highly recommend checking out the dm-tree package: https://github.com/deepmind/tree. It's basically a standalone version of `tf.nest`, which has made my life a lot easier recently. I think rllib already uses that.

Reply:
Ah, good tip. I'll look into this for a later PR...